threads-api/src/threads-api.ts

```diff
@@ -1,6 +1,14 @@
 import axios, { AxiosRequestConfig } from 'axios';
+import { LATEST_ANDROID_APP_VERSION } from 'dynamic-data';
```

Comment:
```suggestion
import { LATEST_ANDROID_APP_VERSION } from './dynamic-data';
```

does it make sence to add `./`?

Reply:
I'll fix this after merging this PR :)